### ISSUE TITLE
Parallel Testing

### ISSUE THREAD

**Issue by Segelzwerg** [Whist-Team/Whist-Server#162]
Is there a way to run the tests in parallel? We recently added quite a few tests and the CI became significantly slower.

I'm not sure if this would help: https://fastapi.tiangolo.com/advanced/async-tests/ ?

**Comment by iTitus** [comment]
I think this is for testing `async` methods and not running tests in parallel

**Comment by Segelzwerg** [comment]
Yes, I guess the problem is the state of test client. Maybe it we use multiple test clients it might work.
